### PR TITLE
Footer kept at the bottom of pages with content

### PR DESCRIPTION
**Pull request by osaosula**
Powered by NextByte at the bottom centre of pages

### CODE REVIEW

File: src/pages/Layout.jsx [hunk 1]
```diff
@@ -27,21 +25,9 @@ const Layout = () => {
...
-            justifyContent: "center",
-            alignItems: "center",
-          }}
-        >
-          Powered by
-          <Link to={"https://www.nextbyteweb.com/"} target="_blank">
-            <img src="./logo-black.webp" alt="NextByte" height={30} />
-          </Link>
-        </Box>
+        <Footer />
```
Comment (BorisLaboratory) [inline src/pages/Layout.jsx:28]: It looks like the Footer should be outside this current Box. It looks like it should be in between line 29 and 30

File: src/pages/Layout.jsx [hunk 2]
```diff
@@ -27,21 +25,9 @@ const Layout = () => {
       <Box component="main" sx={{ flexGrow: 1, p: 3 }}>
         <Toolbar variant="tall" />
         <Outlet />
-        <Box
-          sx={{
-            display: "flex",
-            justifyContent: "center",
```
Comment (BorisLaboratory) [inline src/pages/Layout.jsx:33]: Might be a necessary code in line 27 if you consider moving  Footer to between line 29 and 30

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { Link } from "react-router-dom";
+import { useAuth } from "../hooks/useAuth";
+function Footer() {
+  const { user } = useAuth();
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        justifyContent: "center",
+        position: "relative",
```
Comment (BorisLaboratory) [inline src/components/Footer.jsx:11]: Do you like to consider , position:"fixed" ?

**Review by BorisLaboratory — COMMENTED** [review]
Looks good. Not sure about the functionality though. Are you sure ?

File: src/pages/Layout.jsx (hunk 1 above)
Comment (osaosula) [reply src/pages/Layout.jsx:28]: Yields the same results but you have to change the flex-direction of the enclosing box.
I have done that.

**Review by BorisLaboratory — APPROVED** [review]
I did not request for any changes though. Only needed a response to my comments before approval .

Looks good

File: src/pages/Layout.jsx (hunk 2 above)
Comment (osaosula) [reply src/pages/Layout.jsx:33]: Ok, thanks

**Pull request merged**